pyproject.toml

```
@@ -51,7 +51,7 @@ authors = [
   { name = "Divya Chitimalla" },
   { name = "Katherine Luna" },
 ]
-license = { file = "LICENSE" }
+license-files = ["LICENSE"]
```

Looking at the [guidance for migration](https://packaging.python.org/en/latest/guides/licensing-examples-and-user-scenarios/) to [PEP-639](https://peps.python.org/pep-0639/) format this should be:
```suggestion
license = "Apache-2.0"
license-files = ["LICENSE"]
```

Based on official the [Apache 2.0 reference in SPDX](https://spdx.org/licenses/Apache-2.0.html)

---

I was tempted to add that too but assumed it could pick it out reviewing the file contents. I'm happy to add it to be extra safe :+1: